CMakeLists.txt

```diff
@@ -23,6 +23,9 @@ OPTION(CPUINFO_BUILD_UNIT_TESTS "Build cpuinfo unit tests" ON)
 OPTION(CPUINFO_BUILD_MOCK_TESTS "Build cpuinfo mock tests" ON)
 OPTION(CPUINFO_BUILD_BENCHMARKS "Build cpuinfo micro-benchmarks" ON)
 OPTION(CPUINFO_BUILD_PKG_CONFIG "Build pkg-config manifest" ON)
+OPTION(USE_SYSTEM_LIBS "Use system libraries instead of downloading and building them" OFF)
+OPTION(USE_SYSTEM_GOOGLE_BENCHMARK "Use system Google Benchmark library instead of downloading and building it" ${USE_SYSTEM_LIBS})
```

Comment:
Please use `USE_SYSTEM_GOOGLE_BENCHMARK` for consistency with `GOOGLEBENCHMARK_SOURCE_DIR`

Reply:
@Maratyszcza I guess you've meant `USE_SYSTEM_GOOGLEBENCHMARK`? ;)